### DISCUSSION TITLE
Custom dialog without default dismiss button.

### DISCUSSION THREAD

**Discussion by bidzhao** [fyne-io/fyne#3947] [General]
Is it possible to add new API to create a custom dialog without dismiss button? Because I want to deal with the dismiss logic by providing buttons in its content.

**Comment by andydotxyz** [comment]
I think this is a duplicate of https://github.com/fyne-io/fyne/issues/2127.

This was fixed on develop already for v2.4.0 https://github.com/fyne-io/fyne/pull/3703.

**Reply by bidzhao** [reply]
Hi,

When will v2.4.0 be released?

**Reply by Jacalz** [reply]
Sometime during July probably but August in worst case. Summer is quite a slow month with a lot of people on holiday etc.

**Comment by bidzhao** [comment]
Great! Thanks.
